esmvaltool/cmorizers/data/formatters/datasets/ndp.py

```diff
@@ -13,7 +13,6 @@
     Download the following file:
```

Comment:
This script is currently not working because of the call to the cmorization function.

Reply:
Working on it. By the way, NDP is listed as Tier 3 but I can download it without restriction. Are we sure this is not a Tier 2 dataset?

Reply:
NDP is fixed now and an automatic downloader has been added